tools/readme/readme.md.tmpl

```diff
@@ -173,6 +173,8 @@ The following settings are available per profile:
 * `stack.logstash_enabled` can be set to true to start Logstash and configure it as the
   default output for tests using elastic-package. Supported only by the compose provider.
   Defaults to false.
+* `stack.self_monitor_enabled` enables monitoring and the system package for the managed
+  Elastic Agent. Defaults to false.
```

Comment:
Those settings are assigned to the policy, if another policy is assigned to the agent, the monitoring will depend on the values of the new policy.
Would this be needed or is it redundant ?


```suggestion
* `stack.self_monitor_enabled` enables monitoring and the system package for the default
  policy assigned to the managed Elastic Agent. Defaults to false.
```

Reply:
Well, more clear an specific this way, I will add this, thanks.